scripts/update-apk.sh

```diff
@@ -31,29 +35,29 @@ fi
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	for file in app*; do
-		if [[ $file = "open-event"* ]]; then
+		if [[ $file = "eventyay-attendee"* ]]; then
 			continue
 		fi
-		mv $file open-event-master-${file%%}
+		mv $file eventyay-attendee-master-${file%%}
```

Comment:
Change ${file%%} to ${file:4}

Reply:
Can you please explain?

Reply:
You need to remove the app- part from each apk. Check the issue description

Reply:
We substring it from the 4th index which comes after app-

Reply:
got it

Reply:
Take reference from the linked pr. That's all you need to change

Reply:
> Take reference from the linked pr. That's all you need to change

I've updated it. Can you please review again. It there any way to test the script. When I try to run the script in the build branch, it shows authentication failed.

Reply:
Setup Travi locally, it helps a lot